forms/offer/products/OfferTents.tsx

```diff
@@ -41,7 +40,12 @@ export const OfferTents: FC<IProps> = ({ counties, category, onSubmit }) => {
       .required(t('error.has_transportation.required')),
     town: yup.string(),
     quantity: yup.number().min(1, t('error.quantity.minOne')).typeError(t('error.must.be.number')),
-    tent_capacity: yup.number().required(t('error.tentCapacity.required')),
+    tent_capacity: yup
+      .number()
+      .min(1, t('error.quantity.minOne'))
+      .integer()
```

Comment:
try moving integer() above min() if the validation is not working here

Reply:
aight. will do

Reply:
I'm stupid :)) error prop wasn't passed. fixed now